### PR TITLE
add subdirectory redirect to airtable form for audit program

### PR DESCRIPTION
**Pull request by nearlysean**
add subdirectory redirect to airtable form for audit program 
Was related to ISR-197 but Erik pointed me to this repo. 
After reading on `permanent` true vs false, true seemed better as this is not likely something that will change frequently.

### CODE REVIEW

**Comment by nearlysean** [comment]
Small 4 liner to add a subfolder + redirect.

File: next.config.js [hunk 1]
```diff
@@ -88,6 +88,11 @@ const nextConfig = {
         destination: '/applications?requestAuth=1&createAccount=1',
         permanent: false,
       },
+      {
+        source: '/auditrequest',
+        destination: 'https://airtable.com/appr1nBRRGx2PTJVh/shrpa99vKKW3xafso',
+        permanent: true,
```
Comment (ewiner) [inline next.config.js:94]: I think ’false’ is the better option for this.  Even if we have no plans to change it, permanent redirects can be _very_ permanent, and there's no performance consideration to making sure this URL loads faster by skipping the second lookup.

**Review by ewiner — CHANGES_REQUESTED** [review]

File: next.config.js (hunk 1 above)
Comment (nearlysean) [reply next.config.js]: Thanks I went back and forth after reading the docs. Pushed the change to false. 👍

**Comment by charleslavon** [comment]
thanks for the PR @nearlysean 

Would you mind to also make a similar PR @ https://github.com/near/nearorg. that is the home of the upcoming new near.org

**Review by charleslavon — APPROVED** [review]

**Comment by nearlysean** [comment]
Corresponding PR for nearorg here: https://github.com/near/nearorg/pull/18

**Pull request merged**